docs/en/examples/elementalarea_with_dataobject.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Using ElementalArea with DataObject
+summary: Learn how to add ElementalArea to your models and manage it in a ModalAdmin
+---
+
+# Using ElementalArea with DataObject
+
+## Creating Model and adding ElementalArea
+
+In this example the class BlogPost will be created, it will extend the DataObject class. Also a new BlogPost Admin section will be created in the Admin panel where CMS users can manage the BlogPosts.
```

Comment:
```suggestion
In this example the class `BlogPost` will be created, it will extend the `DataObject` class. Also a new `BlogPost` Admin section will be created in the Admin panel where CMS users can manage the `BlogPost`s.

Note that you don't necessarily need to create an admin section to apply an `ElementalArea` to a `DataObject` - for example you could have a `DataObject` attached to a page.
```
All references to class names, methods, or properties should be quoted with backticks (\`).

Reply:
I'm not sure, that we can split word `BlogPost`s for formatting.